api/src/main/java/com/example/api/service/map/ActivityMapService.java

```diff
@@ -120,6 +121,7 @@ public List<MapTask> getMapTasks(ActivityMap activityMap, User student) {
                 .toList();
         return Stream.of(graphTasks, fileTasks, infos, surveys)
                 .flatMap(List::stream)
+                //.sorted(Comparator.comparingLong(MapTask::getId))
```

Comment:
czemu to jest zakomentowane?

Reply:
nie mam pojęcia co się stało xD, odkomentowane